softdev.bib

```diff
@@ -2871,3 +2871,12 @@ @inproceedings{craciunas08compacting
     month = Jun,
     year = 2008,
 }
+
+@inproceedings{caplinger88memory,
+  author = "Michael A. Caplinger",
```

Comment:
The other entries in the bib do lastname firstname. Should we do this here too?

Reply:
We're all over the shop in that regard. Fortunately, BibTeX treats them as equivalent (except in some unusual cases where you have to use (last, first)).

Reply:
Ok, that works for me then.